base_classes/NXuser.nxdl.xml

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
 <!--
 # NeXus - Neutron and X-ray Common Data Format
-# 
-# Copyright (C) 2008-2022 NeXus International Advisory Committee (NIAC)
-# 
+#
+# Copyright (C) 2014-2024 NeXus International Advisory Committee (NIAC)
```

Comment:
You  cannot do anything with this now, but it is a shame that we overwrite the starting date (in this case from 2008 to 2014).

Reply:
Yeah, dunno why one would have changed that I mean you decided it was 2014, original literature is 2008ish as you correctly point out